src/v2/Components/UserSettings/UserSettingsTabs.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import styled from "styled-components"
+import { RouteTab, RouteTabs } from "v2/Components/RouteTabs"
+import { Box, Text } from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+
+interface UserSettingsTabsProps {
+  route?: string
+  username?: string
+}
+
+const StyledRouteTab = styled(RouteTab)`
+  @media (max-width: ${themeGet("breakpoints.xs")}) {
+    padding-right: 10px;
+  }
+`
+
+const isCurrentTab = (tabUrl, route) => {
+  const currentRoute =
+    typeof window === "undefined" ? route : window.location.pathname
+
+  return currentRoute === tabUrl ? "active" : undefined
```

Comment:
#minor. `isCurrentTab` returns `"active"` class string instead of a boolean seems a bit off.

Reply:
Changed the function name and also made the code more dry: https://github.com/artsy/force/pull/6436/commits/5726c4f284208ba046ec1ecb759c1e12337807f8